GaloyMoney/galoy-mobile
# feat: add account limits screen

**Description**

This PR adds account limits screen to users settings in the app and should be merged after the corresponding [account limits PR](https://github.com/GaloyMoney/galoy/pull/1953) from the Galoy backend repo has been merged.

To test the PR, 
- run the Galoy backend locally and switch to [this](https://github.com/GaloyMoney/galoy/pull/1953) branch
- next, checkout to this commit [`65c649a`](https://github.com/GaloyMoney/galoy/commit/65c649a8ce617f06b4765bed77a4680eeb2accb7) and start the main API. 
- next, in the debugging section of the mobile app, change your backend development server to local and connect your android/iOS simulator/device to the local server (for android run `yarn adb` to sync ports)
- You should find the limits screen under `Settings -> Accounts -> Account Limits`

Update:
- updated the screen to not differentiate USD and BTC transactions.
<img width="384" alt="Screenshot 2022-12-20 at 12 53 15" src="https://user-images.githubusercontent.com/60826700/208662300-266633fe-99d9-487d-9d4b-3bdf6f1a60a3.png">
<img width="372" alt="Screenshot 2022-12-18 at 23 04 56" src="https://user-images.githubusercontent.com/60826700/208322260-9676ea37-39dc-4744-8c33-b577be538ff6.png">
